main/wallet.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const keytar = require('keytar')
+const { generateMnemonic } = require('@zondax/filecoin-signing-tools')
+const { default: Filecoin, HDWalletProvider } = require('@glif/filecoin-wallet-provider')
+const { CoinType } = require('@glif/filecoin-address')
+const electronLog = require('electron-log')
+
+const log = electronLog.scope('wallet')
+
+let address = '' /** @type {string} */
+
+/**
+ * @returns {Promise<string>}
+ */
+async function getSeedPhrase () {
+  const service = 'filecoin-station-wallet'
+  let seed = await keytar.getPassword(service, 'seed')
+  if (seed) {
+    log.info('Using existing seed phrase')
+  } else {
+    seed = generateMnemonic()
+    await keytar.setPassword(service, 'seed', seed)
+    log.info('Created new seed phrase')
+  }
+  return seed
+}
+
+async function setup () {
+  const seed = await getSeedPhrase()
+  const provider = new Filecoin(new HDWalletProvider(seed), {
+    apiAddress: 'https://api.node.glif.io/rpc/v0'
+  })
+  ;[address] = await provider.wallet.getAccounts(
+    0,
+    1,
+    CoinType.MAIN
+  )
+  log.info('Address: %s', address)
+}
+
+/**
+ * @returns {string}
+ */
+function getAddress () {
```

Comment:
When reading code calling `wallet.getAddress()`, I am not sure whether it's the address of the local wallet or the destination address configured in the Wallet UI.

I think a name like `getStationAddress()` or `getLocalAddress` may work better.

Reply:
Hm, I'm surprised by this, since to me `wallet` by definition is the only wallet available to Station. And that is the internal one. And a wallet only has one address, so `wallet.getStationAddress()` doesn't make so much sense to me. If anything I think we could rename it to `stationWallet.getAddress()`, however that's prefixing our own components with our own name, which is kind of redundant. What do you think?

Reply:
I see. Your explanation makes sense 👍🏻 Let's keep things as they are now.